controller/src/pid.cpp

```diff
@@ -116,16 +121,33 @@ void pid_execute(const VentParams &params, SensorReadings *readings) {
     break;
   }
 
+  //@TODO: The following are raw accesses that need to be updated to reference
+  // the new modules because currently they do not have the benefits of things
+  // like calibration, etc. that the module methods do. I didn't do this because
+  // I have no clue what the desired units are for the control system.
   // Update PID Loop
   int16_t sensorValue = Hal.analogRead(DPSENSOR_PIN); // read sensor
-  Input = map(sensorValue, 0, 1023, 0, 255);  // map to output scale
-  myPID.Compute();                            // computer PID command
+  // int16_t sensorValue = get_pressure_reading(PressureSensors::SomeDPPin);
+  Input = map(sensorValue, 0, 1023, 0, 255); // map to output scale
+  myPID.Compute();                           // computer PID command
   Hal.analogWrite(BLOWERSPD_PIN, static_cast<int>(Output)); // write output
 
   // Store sensor readings so they can eventually be sent to the GUI.
-  // TODO(jlebar): Is sensorValue actually pressure in cm of h2o?
   // TODO(jlebar): Get and store volume and flow data.
-  readings->pressure_cm_h2o = sensorValue;
+  // TODO(jlebar/verityRF): determine what volume_ml and flow_ml_per_min really
+  // means and how that is calculated from the two different venturi tubes in
+  // the air loop. Most likely need to have an integrator for flow to get the
+  // volume_ml.
+  // TODO(someone): is pressure_cm_h2o just the patient pressure sensor
+  // converted to the right unit? Convert [kPa] to [cm H2O] in place
```

Comment:
It's up to us what we want those readings numbers to mean -- that was my first attempt -- but you're interpreting all of them the way I expected!  If any of the descriptions in the .proto file are unclear, would you be willing to add a comment that clears it up for you?

Reply:
All good -  See above conversation. I would add the comments if I knew what these meant :)
All I'm saying is, I don't have the current knowledge to know what these parameters truly relate to in terms of the real air circuit (are they direct from 1 of the 3 possible sensors, or some derived value from some combination of them?). Not a hard problem, someone here definitely knows, but I don't!

Reply:
> someone here definitely knows

lol you have more faith than I do

Can we at least write the question as concretely as we can?  That'll help us get an answer.

Reply:
I'll post it in the slack and see what we get, can reference it in the actual issue.